docs/features/templates.rst

```diff
@@ -18,17 +18,27 @@ will ignore the bundled one.
 Templates set
 *************
 
-By using **Apphook configuration** you can define a different templates set.
-To use this feature provide a directory name in **Template prefix** field in
-the **Apphook configuration** admin (in *Layout* section): it will be the
-root of your custom templates set.
+You can provide different set of templates for the whole ``djangocms-blog`` application by configuring
+the ``Blog configs`` accordingly.
+
+This require you to customize **all** the templates shipped in ``djangocms_blog/templates/djangocms_blog``, the easiest
+way being copy the **content** of ``djangocms_blog/templates/djangocms_blog`` in another folder in the ``templates``
+folder in our project
+(e.g., something like ``cp -a djangocms_blog/templates/djangocms_blog/* /path/my/project/templates/my_blog``).
+
+To use the new templates set, go to the ``Blog configs`` admin
+(something like ``http://localhost:8000/en/admin/djangocms_blog/blogconfig/1/change``) and enter a directory name in
+**Template prefix** field in the **Apphook configuration** admin (in *Layout* section): it will be the
+root of your custom templates set; following the example above you should enter ``my_blog`` as directory name.
```

Comment:
Those first two statements are golden; they would be very helpful. As a suggestion, I've merely tweaked some grammar and punctuation.

```suggestion
You can provide a different set of templates for the whole ``djangocms-blog`` application by configuring
the ``Blog configs`` accordingly.

This would require you to customize **all** the templates shipped in ``djangocms_blog/templates/djangocms_blog``; the easiest
way would be to copy the **content** of ``djangocms_blog/templates/djangocms_blog`` into another folder in the ``templates``
folder in our project
(e.g., something like ``cp -a djangocms_blog/templates/djangocms_blog/* /path/my/project/templates/my_blog``).

To use the new templates set, go to the ``Blog configs`` admin
(something like ``http://localhost:8000/en/admin/djangocms_blog/blogconfig/1/change``) and enter a directory name in the
**Template prefix** field in the **Apphook configuration** admin (in the *Layout* section): it will be the
root of your custom templates set; following the example above, you should enter ``my_blog`` as directory name.
```

I propose a final paragraph, so you do not need to revisit this for ignorant* developers [official documentation that subject](https://docs.djangoproject.com/en/3.0/howto/overriding-templates/):

> For more instruction, please read [Django (3.0): Overriding templates](https://docs.djangoproject.com/en/3.0/howto/overriding-templates/#overriding-templates) (for your version of Django).

P.S. I do not know `.rst` (and am too tired, today, to [learn](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links)).

* I.e. "lacking knowledge". No offense intended. I have been an "ignorant developer" all week.

Reply:
we are all ignorant developers at something at any given time :)